Libraries/Vibration/Vibration.js

```diff
@@ -27,11 +27,31 @@ var Platform = require('Platform');
  */
 
 var Vibration = {
-  vibrate: function(duration: number = 400) {
+  vibrate: function(pattern: number | Array<number> = 400, repeat: number = -1) {
```

Comment:
http://developer.android.com/intl/ru/reference/android/os/Vibrator.html#vibrate(long[], int)
```repeat``` the index into pattern at which to repeat, or -1 if you don't want to repeat.
```true | false``` is easier api but we will lose ability to repeat since exact index.